jdaviz/configs/default/plugins/export/export.py

```diff
@@ -187,6 +190,22 @@ def _sync_singleselect(self, event):
                 setattr(self, attr, '')
             if attr == 'subset_selected':
                 self._set_subset_not_supported_msg()
+            elif attr == "viewer_selected" and self.config == "cubeviz":

```

Comment:
Should we swap the conditions here so all the other viz doesn't have to check for attr to begin with?